doc/api/esm.md

```diff
@@ -732,9 +732,9 @@ the chain.
 changes:
   - version: REPLACEME
     pr-url: https://github.com/nodejs/node/pull/42623
-    description: Add support for chaining resolve hooks. This introduces a
-      breaking change, wherein a hook that does not call `nextResolve()` MUST
-      include a `shortCircuit` flag in its return.
+    description: Add support for chaining resolve hooks. Each hook that does
+      not call `nextResolve()` MUST include a `shortCircuit` property set to
```

Comment:
I think it's a weird way of presenting the options, why not go the `must either` route? Why is must in capital letters?

Reply:
since there's no markdown to emphasise the text

Reply:
I don't think the emphasis is needed.